membership-attribute-service/app/services/SupporterProductDataService.scala

```diff
@@ -0,0 +1,45 @@
+package services
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBAsync
+import com.gu.scanamo.error.DynamoReadError.describe
+import com.gu.scanamo.syntax._
+import com.gu.scanamo.{ScanamoAsync, Table, _}
```

Comment:
that's a strange import, woudln't `_` on its own be the same?

Reply:
Yup, Intellij having fun